domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/MediationRow.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import lombok.Builder;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MediationRow {
+    public static final String SITE_ID = "4";
```

Comment:
please add equalandhasocde annotation for Lombok

Reply:
I have added it. Probably a basic question, is that really necessary since I am never going to check/use if two MediationRow objects are equal?

Reply:
Just a question of good practice. What it means for an object to be equal to another is part of the class definition.